Swift/ConsoleLogger.swift

```diff
@@ -20,6 +20,33 @@
 import Foundation
 
 /// Console logger for writing log messages to the system console.
+
+/// Log domain options that can be enabled in the console logger.
+public struct LogDomains: OptionSet {
+    /// Raw value.
+    public let rawValue: Int
+    
+    /// Constructor with the raw value.
+    public init(rawValue: Int) {
+        self.rawValue = rawValue
+    }
+    
+    /// All domains.
+    public static let all = LogDomains(rawValue: Int(LogDomain.all.rawValue))
```

Comment:
```suggestion
    public static let all =[.database, .query, .replicator, .network]
```

Reply:
I want to keep the value inline with LogDomain.all and objc value. So it’s better to keep it that way.